osquery/tables/system/windows/users.cpp

```diff
@@ -24,81 +24,138 @@
 #include "osquery/core/conversions.h"
 
 namespace osquery {
+
+std::string psidToString(PSID sid);
```

Comment:
If it doesn't exist, this isn't enough to force that refactor. The utilities that table use and other parts of the code case use can remain scattered. But there are utilities that are isolated to table use that need another design decision (like yara).